### PR TITLE
AIL 96/feature/allow multiple llm usage

### PR DESCRIPTION
**Pull request by ninad-mohite-lrn**

### CODE REVIEW

File: utils/aws_bedrock.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from langchain_community.chat_models import BedrockChat
```
Comment (sean-mccrossan-lrn) [inline utils/aws_bedrock.py:1]: Should this be in the general utils folder? Seems to be specific to language translation?

File: bias-detection/utils/s3_manager.py
Comment (sean-mccrossan-lrn) [inline bias-detection/utils/s3_manager.py:1]: Would move to the general utils folder.

**Review by sean-mccrossan-lrn — APPROVED** [review]
Made a few comments but lg to me. Think poetry is a good move and making it easier to initialise models.

File: utils/aws_bedrock.py (hunk 1 above)
Comment (ninad-mohite-lrn) [reply utils/aws_bedrock.py:1]: I think its in genrral utils folder. It in the ml-flow-eval/utils

File: bias-detection/utils/s3_manager.py
Comment (ninad-mohite-lrn) [reply bias-detection/utils/s3_manager.py:1]: I just kept your stuff untouched. Just moved to one folder

File: utils/aws_bedrock.py (hunk 1 above)
Comment (ninad-mohite-lrn) [reply utils/aws_bedrock.py:1]: ohh sorry I misunderstood. I'll make it generic, or best would be to get it from the user.

File: utils/openai.py
Comment (sean-mccrossan-lrn) [inline utils/openai.py:1]: LG

File: utils/aws_bedrock.py
Comment (sean-mccrossan-lrn) [inline utils/aws_bedrock.py:1]: LG

**Review by sean-mccrossan-lrn — APPROVED** [review]
All good thanks

**Pull request merged**